app/assets/stylesheets/mortgage_calculator/components/_risk.scss

```diff
@@ -35,104 +35,44 @@
   display: none;
 }
 
-.risk--high {
-  .risk__accent--color-only,
-  .risk__accent {
-    color: $colour-risk-high;
+$risk-levels: (high, $colour-risk-high) (medium, $colour-risk-medium) (low, $colour-risk-low);
 
-    .ui-slider-handle {
-      &:focus,
-      &:active {
-        border-color: $colour-risk-high;
-      }
-    }
-
-    .ui-slider-handle {
-      &:after {
-        border-bottom-color: $colour-risk-high;
-      }
-    }
-  }
+@each $level in $risk-levels {
 
-  .risk-slice--incoming {
-    fill: $colour-risk-high;
-  }
+  $level-name: nth($level, 1);
+  $level-color: nth($level, 2);
 
-  .risk-key--incoming {
-    &:after {
-      background: $colour-risk-high;
+  .risk--#{$level-name} {
+    .risk__accent--color-only {
+      background-color: $level-color;
```

Comment:
Yep that's right. Using coloured text on a white background is low contrast, so we are using black text throughout and just changing the background colour instead.